### PR TITLE
feat(component folder): customComponent for Button,Input

### PR DESCRIPTION
**Pull request by gautism1**
Made all the changes those were asked to do ,

### CODE REVIEW

File: src/pages/Invoice/CreateInvoice.js [hunk 1]
```diff
@@ -187,7 +188,7 @@ const CreateInvoice = () => {
                                         {" "}
                                         <FormattedMessage id="invoice.issuedAt" />
                                     </label>
-                                    <input
+                                    <Input
                                         className="input-sm"
```
Comment (rparsh) [inline src/pages/Invoice/CreateInvoice.js:192]: make new props `size` can support `sm` | `md` for now if `lg` require will introduce later

Comment (rparsh) [reply src/pages/Invoice/CreateInvoice.js:192]: `md` being default

Comment (gautism1) [reply src/pages/Invoice/CreateInvoice.js]: introduces size props

Comment (gautism1) [reply src/pages/Invoice/CreateInvoice.js]: make md as default class

File: src/pages/Invoice/InvoiceModal.js [hunk 2]
```diff
@@ -3,6 +3,8 @@ import PropTypes from "prop-types";
 import Modal from "react-modal";
 import { Link } from "react-router-dom";
 import { FormattedMessage } from "react-intl";
+import Button from "components/Button";
+
```
Comment (smilegupta) [inline src/pages/Invoice/InvoiceModal.js:7]: Why leaving extra line empty here?

File: src/pages/Invoice/InvoiceModal.js [hunk 3]
```diff
@@ -139,9 +141,9 @@ const ChangeCustomerModal = ({
                                                 <p>Item: {info.name}</p>
                                                 <p>Price: ₹{info.price}</p>
                                             </div>
-                                            <button className="btn">
+                                            <Button>
                                                 <FormattedMessage id="select"></FormattedMessage>
-                                            </button>
+                                            </Button>
```
Comment (smilegupta) [inline src/pages/Invoice/InvoiceModal.js:146]: Please use <FormattedMessage /> kind of str in the code to make it uniform throughout

File: src/components/card.css [hunk 4]
```diff
@@ -0,0 +1,4 @@
+.card-default {
+    padding: 1rem;
+    background-color: white;
+}
```
Comment (smilegupta) [inline src/components/card.css:4]: Why is the need to make card-default class? when we have card property added in index.css.
Also what is the need for card.css. We can include all generic style properties in index.css

**Review by smilegupta — COMMENTED** [review]
Added Some Comments

File: src/components/card.css (hunk 4 above)
Comment (gautism1) [reply src/components/card.css:4]: it was mentioned to have these two styles in issue ,ok i will include its in index.css

**Review by smilegupta — APPROVED** [review]

File: src/components/Button.js [hunk 5]
```diff
@@ -0,0 +1,19 @@
+import React from "react";
+import Proptypes from "prop-types";
+
+function Button({ type = "button", children, icon, ...rest }) {
+    return (
+        <button type={type} className="btn" {...rest}>
```
Comment (rparsh) [inline src/components/Button.js:6]: className prop support missing

**Pull request merged**